api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -162,4 +169,22 @@ public void deleteWorkspace(String projectName, String workspaceName) throws Api
     WorkspacesApi workspacesApi = workspacesApiProvider.get();
     workspacesApi.deleteWorkspace(projectName, workspaceName);
   }
+
+  @Override
+  public ManagedGroupWithMembers createRegisteredGroup() throws ApiException {
+    GroupsApi groupsApi = groupsApiProvider.get();
+    return groupsApi.createGroup(REGISTERED_GROUP_NAME);
+  }
+
+  @Override
+  public void addUserToRegisteredGroup(String email) throws ApiException {
```

Comment:
Eventually we're going to have to be able to add / remove users from the "controlled" authorization domain as well. (See other comments.) So I think these methods should probably change to:

createGroup(String)
addUserToGroup(String, String)
removeUserFromGroup(String, String)

And then have clients use WorkbenchConfig to look up the appropriate group name for the context.